### ISSUE TITLE
Uncaught ReferenceError: parcelRequire is not defined

### ISSUE THREAD

**Issue by RedRockerSE** [jshor/datebook#64]
I'm trying to add the package to a Angular 8-project but adding code as the example on the README makes the site note render and the console gives this error:

```
index.js:1 Uncaught ReferenceError: parcelRequire is not defined
    at push../node_modules/datebook/dist/index.js.parcelRequire.a2/B (index.js:1)
```

Any tips?

**Comment by TonyDeplanque** [comment]
Same issue. How fixed it ?

**Comment by jshor** [comment]
This is now fixed as of v4.0.0.
